index.js

```diff
@@ -40,7 +46,22 @@
 // console.log(sameSameButDifferent(5, 5)) 
 // // => "same same"
 
-// console.log(sameSameButDifferent(123, "123")) 
+function sameSameButDifferent(num,maybeNum){
+  
+  if (num === maybeNum)
+  {
+    return "same same";
+  }
+  else if (num == maybeNum)
+  {
+    return "same same but different"
+  }
+  else
+  {
+    return "Different."
+  }
+  }
+// console.log(sameSameButDifferent(123, '123')) 
```

Comment:
Perfect. You could also try playing around with parseInt in the else if and avoid having to use ==

you could also dig deeper and use typeof to check the data type of the variable
if (typeof num === 'number' && typeof maybeNum === 'number' && num === maybeNum)